en/docs/cli/init.md

```diff
@@ -83,6 +83,16 @@ success Saved package.json
 ✨  Done in 121.53s.
 ```
 
+##### Setting defaults for  `yarn init` <a class="toc" id="toc-yarn-init-defaults" href="#toc-yarn-init-defaults"></a>
+
+The following [config](./config.md) variables can be used to customize the defaults for `yarn init`:
```

Comment:
This link doesn't work properly - I think you need to just link to `config` without the `.md` extension.

Take a look at the site preview here: https://deploy-preview-289--assembler-pig-23305.netlify.com/en/docs/cli/init (this shows what your changes would look like on the site)